internal/model/manifest.go

```diff
@@ -20,6 +20,7 @@ type Manifest struct {
 	DockerRef  reference.Named
 
 	// Local files read while reading the Tilt configuration.
+	ConfigFiles []string
```

Comment:
this seems redundant with ConfigMatcher? it seems like we should only have one

Reply:
We need to know the names of the config files in order to watch them. They could be anywhere on disk, and aren't guaranteed to be in the working directory with the Tiltfile (as is the case with Servantes).

Reply:
ya, tho we should probably get rid of the matcher as a separate field then